docs/scheduled-transactions/revised-spec.md

```diff
@@ -0,0 +1,149 @@
+
+#### Duplicate creations
+There is a special case in which a `ScheduleCreate` transaction tries to re-create a
+schedule that already exists in state. When this happens, the `ScheduleCreate` resolves 
+to a new status of `IDENTICAL_SCHEDULE_ALREADY_CREATED`, and the `ScheduleID` in its receipt points
+to the existing schedule. A client receiving `IDENTICAL_SCHEDULE_ALREADY_CREATED` can
+then submit a `ScheduleSign` (see below) with the given `ScheduleID`, signing with
+the same Ed25519 keys it used for its own create attempt. 
+
+Two <tt>ScheduleCreate</tt> transactions are <i>identical</i> if they are equal in all their fields 
+other than, possibly, <tt>payerAccountID</tt>. (Here "equal" should be understood in the sense of 
+gRPC object equality in the network software runtime. In particular, a gRPC object with 
+[unknown fields](https://developers.google.com/protocol-buffers/docs/proto3#unknowns)
+is not equal to a gRPC object without unknown fields, even if they agree on all known fields.)
+  
+### Enforced checks
+
+The only body-specific precheck enforced for a `ScheduleCreate` transaction is that the 
+`memo` field is valid. At consensus, the following checks are enforced:
+  1. The `memo` must be valid.
+  2. The type of the `scheduledTransactionBody` must be in the `scheduling.whitelist`; and it 
+     must reference only non-deleted entities that exist in state at the time the `ScheduleCreate` 
+     reaches consensus. 
+  3. The `adminKey` must be valid, if present.
+  
+## The `ScheduleSign` transaction
+  
+```  
+message ScheduleSignTransactionBody {  
+  ScheduleID scheduleID = 1; // The id of an existing schedule to affirm signing keys for
+}  
+```  
+
+Suppose we use the `scheduleID` field to point to schedule `0.0.12345`, which contains a 
+scheduled transaction `X`. If any of the Ed25519 keys required to sign `X` sign 
+our `ScheduleSign` ordinary transaction, then the network will update the state of 
+schedule `0.0.12345` with these keys.
+
+(It is permissible, though pointless, to sign the `ScheduleSign` with keys not 
+required for the execution of `X`.)
+
+### Receipts
+
+When a `ScheduleSign` resolves to `SUCCESS`, its receipt includes a new 
+`scheduledTransactionID` field which is the `TransactionID` that can be used to query 
+for the record of the execution of the scheduled transaction in the signed schedule.
+  
+## The `ScheduleDelete` transaction
+  
+```  
+message ScheduleDeleteTransactionBody {  
+  ScheduleID scheduleID = 1; // The id of an existing schedule to delete
+}  
+```  
+  
+## The `ScheduleGetInfo` query
+  
+```  
+message ScheduleGetInfoQuery {  
+  QueryHeader header = 1; // Standard query metadata, including payment
+  ScheduleID schedule = 2; // The id of an existing schedule
+}  
+  
+message ScheduleGetInfoResponse {  
+  ScheduleID scheduleID = 1; // The id of the schedule
+  bool deleted = 2; // Has the schedule been deleted? 
+  bool executed = 3; // Has the schedule been executed? 
```

Comment:
What about instead of a boolean a consensus `Timestamp` at which it was executed? This conveys the same info as a boolean (it's executed if present) with additional information about when it was executed.

Reply:
(Above.)